web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -2051,11 +2045,10 @@ private BaseQueryResponse next(final String id, boolean checkForContentLookup) {
                 log.error("Error committing transaction: resources rolled back transaction", e);
             } catch (Exception e) {
                 log.error("Error committing transaction: Unknown error", e);
-            } finally {
-                // Stop timing on this trace, if any
-                if (span != null) {
-                    span.stop();
-                }
+            }
+            // Stop timing on this trace, if any
+            if (span != null) {
+                span.stop();
```

Comment:
I'm curious as to why this was moved outside of the finally block? Don't we want to stop the "next" call span even if the call itself fails with an exception?

Reply:
In this case the enclosing scope is already a _finally_ block, starting around L2029, so the extra _finally_ didn't seem to buy us much here, since any exceptions will still be suppressed above. I suppose the span still might not be stopped here, if some java.lang.Error-derived Throwable is raised in the preceding _try_. Moving this block above the _try_ would mitigate that though. I'll do that